src/acsets/acsets.py

```diff
@@ -251,7 +250,7 @@ def __init__(
             attrs: A list of attributes (`Attr`).
         """
         self.name = name
-        self.schema = CatlabSchema(VERSION_SPEC, obs, homs, attrtypes, attrs)
```

Comment:
It's because the `VERSION_SPEC` is set in the init function and that parameter doesn't actually exist:

https://github.com/AlgebraicJulia/py-acsets/pull/21/files#diff-5aea4dbcecaaaf6020fdcd5941624591d5b6d0c8bd8da0edab40aaef29f17dbbR206-R212

Reply:
maybe what should actually be happening is we fix the init function? Is that what you mean?

Reply:
Oh gotcha. Yeah, that's fine then.